src/components/VulnerabilityEdgePrompt.js

```diff
@@ -1,39 +1,174 @@
+  // useEffect(() => {
+  //   !CVEs?.isLoading &&
+  //     !CVEs?.meta?.filter &&
+  //     setUpdateCveModal((prevState) => ({
+  //       ...prevState,
+  //       cveCount: CVEs?.data?.length,
+  //     }));
+  // }, [CVEs, deviceData, newImageStatus, imageId]);
+
+  const alerts = {
+    updateImage: (
+      <Alert
+        className="pf-u-mb-md"
+        variant="info"
+        isInline
+        title="To remediate CVEs, update the image."
+        actionLinks={
+          <Button
+            className="pf-u-mt-sm"
+            isSmall
+            onClick={(e) => {
+              e.preventDefault();
+              window.location.href = `insights/image-builder/manage-edge-images/${imageSetId}?update_image=true`;
+            }}
+          >
+            Update Image
+          </Button>
+        }
+      />
+    ),
+    imageBuilding: (
+      <Alert
+        className="pf-u-mb-md"
+        customIcon={<InProgressIcon />}
+        variant="info"
+        isInline
+        title="Image build in progress. Once completed, you'll need to update your system."
+      />
+    ),
+    updateDevice: (
+      <Alert
+        className="pf-u-mb-md"
+        variant="warning"
+        isInline
+        title=" Image build completed. Update system to the newest image version to remediate CVEs."
+        actionLinks={
+          <Button
+            className="pf-u-mt-sm"
+            isSmall
+            onClick={(e) => {
+              e.preventDefault();
+              window.location.href = `insights/inventory/${inventoryId}/update`;
+            }}
+          >
+            Update system
+          </Button>
+        }
+      />
+    ),
+    systemUpdating: (
+      <Alert
+        className="pf-u-mb-md"
+        customIcon={<InProgressIcon />}
+        variant="info"
+        isInline
+        title="System updating. No additional actions required."
+      />
+    ),
+    noAlert: '',
+  };
+
   return (
-    <EmptyState data-cy="vulnerability-edge-prompt">
-      <EmptyStateIcon icon={MonitoringIcon} />
-      <Title headingLevel="h4" size="lg">
-        View and triage CVEs in Edge Management
-      </Title>
-      <EmptyStateBody>
-        To view and triage CVEs, update the image in the Edge Management
-        application
-      </EmptyStateBody>
-      <Button
-        component="a"
-        href={`${
-          chrome.isBeta() ? '/preview' : ''
-        }/edge/inventory/${inventoryId}`}
-        variant="primary"
-      >
-        Go to Edge Management
-      </Button>
-    </EmptyState>
+    <section className="add-100vh pf-l-page__main-section pf-c-page__main-section">
+      {alerts[activeAlert]}
+      <AsyncComponent
+        appName="vulnerability"
+        module="./SystemDetail"
+        getRegistry={getRegistry}
+        customIntlProvider
+        entity={{ id: inventoryId }}
+        canSelect={false}
+        canEditPairStatus={false}
+        canManageColumns={false}
+        linkToCustomerPortal
+        defaultColumns={[
```

Comment:
This is more of a nitpick, but we typically dont set columns like this in the prop. We define it in a variable and then slap that variable in the prop. Something like 
`defaultColumns={builtColumns}`